src/EnergyPlus/ThermalComfort.cc

```diff
@@ -715,6 +718,16 @@ namespace ThermalComfort {
 			} else {
 				ShowSevereError( "PEOPLE=\"" + People( PeopleNum ).Name + "\", Incorrect Clothing Type" );
 			}}
+			// for unit testing
```

Comment:
This seemed like a low impact way of setting some parameters to use when testing the routine. It does not affect any other code since these are optional arguiments.